lib/mix/tasks/phx.gen.release.ex

```diff
@@ -143,6 +148,18 @@ defmodule Mix.Tasks.Phx.Gen.Release do
     """)
   end
 
+  defp parse_args(args) do
+    args
+    |> OptionParser.parse!(strict: [ecto: :boolean, docker: :boolean])
+    |> elem(0)
+    |> Keyword.put_new_lazy(:ecto, &ecto_sql_installed?/0)
+    |> Keyword.put_new(:docker, false)
+    |> Map.new(fn
+      {:ecto, val} -> {:ecto?, val}
+      {:docker, val} -> {:docker?, val}
+    end)
```

Comment:
```suggestion
    |> Map.new()
```

Reply:
Of course this requires changing the rest of the code but given that we don't rewrite options like this anywhere else, I would skip this. :)